src/main/java/com/openlattice/datastore/apps/services/AppService.java

```diff
@@ -240,6 +240,21 @@ public void installApp( UUID appId, UUID organizationId, String prefix, Principa
         organizationService.addAppToOrg( organizationId, appId );
     }
 
+    public void uninstallApp( UUID appId, UUID organizationId ) {
```

Comment:
I'm not sure about the role of the entity set created, when an app is installed, just double checking, if that needs to be handled here or not.

Reply:
yeah I figured it's safer to let the user delete the entity sets if they want, since it's possible they don't want to lose all their data but just don't want to use the app anymore